sourceCode.py

```diff
@@ -1 +1,523 @@
+
+## hello reviewer, this is by the way the main file to review. Ill be grateful about some tips on how to integrate everything and how to create a web interface thingy
+## I have now created some libraries per topic and functions as you can see, but did not integrate any grammar stuff yet or any more general construct in which the stuff will be embedded in, like choosing a topic. will do that next time. tired now. 
+
+##########################################################
+##                                                      ##
+## grammar lesson 1: first person becomes second person ##
+##                                                      ##
+##########################################################
+
+gReflections = {
+    "am"   : "are",
+    "was"  : "were",
+    "i"    : "you",
+    "i'd"  : "you would",
+    "i've"  : "you have",
+    "i'll"  : "you will",
+    "my"  : "your",
+    "are"  : "am",
+    "you've": "I have",
+    "you'll": "I will",
+    "your"  : "my",
+    "yours"  : "mine",
+    "you"  : "me",
+    "me"  : "you"
+    }
+  
+                                ########################################################################################  
+                                ##   -------------------- Topic Functions and Libraries -------------------------     ##
+                                ########################################################################################
+
+
+
+##############################
+##                          ##
+## small talk with Jordan   ##
+##                          ##
+##############################
+
+## create the library for small-talk with Jordan
+library_smalltalk = [
+    [r'Hi(.*)',
+     [  "Hi, how are you?",
+      "Hey, whazzup?",
+      "Hello you!"]],
+    
+    [r'How are you?',
+     [  "I am fine, thanks. You?",
+      "Great, you?",
+      "I am doing wonderful. How are you doing?"]],
+    
+    [r'I am (.*)',
+     [  "That is nice to hear!",
+      "How wonderful."]],
+    
+    [r'I want (.*)',
+     [  "Yeah, me too!",
+      "Why do you want %1?",
+      "What would you do if you got %1?",
+      "If you got %1, then what would you do?"]],
+    
+    [r'It is (.*)',
+     [  "You seem very certain.",
+      "Interesting. Why?"]],
+    
+    [r'Because (.*)',
+     [  "I see."]],
+    
+    [r'(.*) weather(.*)',
+     [  "I love talking about the weather.",
+      "Does it seem strange to talk to a computer about the weather?",
+      "It is super sunny today, right?"]],
+    
+    [r'quit',
+     [  "Thank you for talking with me.",
+      "Good-bye!.",
+      "Have a nice day!"]],
+    ]
+    
+## create the function to have some small-talk with Jordan
+
+import random
+import re
+
+def smalltalk_Jordan(userInput):
+    keys_smalltalk= list(map(lambda x:re.compile(x[0], re.IGNORECASE), library_smalltalk))
+    values_smalltalk = list(map(lambda x:x[1],library_smalltalk))
+    
+    while True:
+        userInput = input(">>> ")
```

Comment:
I would do something like userInput = input("I'm the Jordan Help Bot! I'm looking forward to chat with you") or something to make it more interactive and initiate the user write something :) but that's a minor thing